### PR TITLE
Format code and replace var declarations with 

### PR DESCRIPTION
**Pull request by pavankjadda**
1. Replace var declarations with let
2. Add default language to HTML pages

### CODE REVIEW

**Comment by iamadamdev** [comment]
I follow Google's style guide of using 2 spaces for indentation.

**Comment by pavankjadda** [comment]
I used Webstorm default indention. We use it in Angular project at my work. I will check Google style guide.

**Comment by pavankjadda** [comment]
Okay. I formatted code with Google style guide. It should be good now

**Comment by iamadamdev** [comment]
Rebase off latest master please.

Standard looks good, I might start using that. By default doesn't it also enforce 2 space indentation? Actually it's weird, on some files (like common.js) it says "Expected indentation of 8 spaces but found 4." and others it says 2.

Do you have a custom config file?
It also wants to remove the trailing semicolons but in your PR I still see them there.
I also prefer to keep trailing semicolons.

**Comment by pavankjadda** [comment]
I used downloaded Google Code style from one of the Github repo and formatted the code.

[GoogleStyle.xml.zip](https://github.com/iamadamdev/bypass-paywalls-chrome/files/4526455/GoogleStyle.xml.zip)

Actually I do like semicolons at the end(started programming with C, so keep in using it). After importing the config file/style guide, you can manually change the settings. Anyway, I will rebase the master branch and format the code with Standard JS. Will that be okay?

File: src/js/background.js [hunk 1]
```diff
@@ -412,44 +432,50 @@ extension_api.webRequest.onBeforeSendHeaders.addListener(function(details) {
     extension_api.tabs.executeScript(tabId, {
       file: 'contentScript.js',
       runAt: 'document_start'
-    }, function(res) {
+    }, function (res) {
       if (extension_api.runtime.lastError || res[0]) {
-        return;
```
Comment (iamadamdev) [inline src/js/background.js:417]: Unexpected change?

File: src/js/sites.js [hunk 2]
```diff
@@ -1,133 +1,132 @@
...
-    'The Seattle Times': 'seattletimes.com',
-    'The Spectator Australia': 'spectator.com.au',
-    'The Spectator': 'spectator.co.uk',
-    'The Sydney Morning Herald': 'smh.com.au',
-    'The Telegraph': 'telegraph.co.uk',
-    'The Times': 'thetimes.co.uk',
-    'The Toronto Star': 'thestar.com',
-    'The Wall Street Journal': 'wsj.com',
-    'The Washington Post': 'washingtonpost.com',
-    'The Wrap': 'thewrap.com',
```
Comment (iamadamdev) [inline src/js/sites.js:124]: I don't see this site being re-added later

**Comment by pavankjadda** [comment]
I will go ahead and close this then

**Comment by pavankjadda** [comment]
@iamadamdev And you might want to replace those `var` declarations with `let` and limit the scope of the variables outside of the function/block.

**Pull request closed without merging**